kedro_airflow_k8s/airflow_dag_template.j2

```diff
@@ -297,6 +297,11 @@ with DAG(
 
     {% if not config.run_config.volume.disabled %}
     kedro_group >> delete_pipeline_storage
+
+    from airflow.operators.dummy import DummyOperator
+    final = DummyOperator(task_id='final', dag=dag)
+    kedro_group >> final
+    delete_pipeline_storage >> final
```

Comment:
Dependency is required for tasks["delete-spark-cluster"]  if present as well.

Reply:
Added.